tests/integration_tests/api/auth_methods/test_ldap.py

```diff
@@ -15,12 +16,21 @@ class TestLdap(HvacIntegrationTestCase, TestCase):
 
     @classmethod
     def setUpClass(cls):
-        super(TestLdap, cls).setUpClass()
-        logging.getLogger('ldap_test').setLevel(logging.ERROR)
+        # The mock LDAP server requires Java runtime
+        if not distutils.spawn.find_executable('java'):
+            raise SkipTest
```

Comment:
This comment could be migrated to the exception instance being raised, no? It would still exist for folks looking 
```suggestion
            raise SkipTest('The mock LDAP server requires Java runtime')
```

Reply:
I'm not sure why I didn't think of that.  It should be an easy fix.